src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enums in Java should be singular. The thinking is that you're not selecting multiple but one of the possible choices in the list of values. Rename 'Colors' to 'Color'